js/actions.js

```diff
@@ -2,7 +2,7 @@
  * Sample actions execution file
  */
 
-(function ( $, window, document, Motif, undefined ) {
+(function ( $, window, document, Modernizr, Motif, undefined ) {
```

Comment:
This removes the undefined, but it just assigns it to the 4th parameter on Line 46, which is `window.Motif`. I don't think this is complete until we add `window.Modernizr` (or something) as the 4th parameter at the bottom.